attributes/default.rb

```diff
@@ -64,6 +66,9 @@
 default['cfncluster']['efa']['installer_url'] = 'https://s3-us-west-2.amazonaws.com/aws-efa-installer/aws-efa-installer-1.5.4.tar.gz'
 # ENV2 - tool to capture environment and create modulefiles
 default['cfncluster']['env2']['url'] = 'https://sourceforge.net/projects/env2/files/env2/download'
+# CloudWatch Agent
+default['cfncluster']['cloudwatch']['public_key_url'] = 'https://s3.amazonaws.com/amazoncloudwatch-agent/assets/amazon-cloudwatch-agent.gpg'
```

Comment:
we need at least one URL for the China

Reply:
I've made the changes to the URLs. I am in the process of creating an account in the China partition to test them out there.

Reply:
For me, the un-regionalized URL (used for downloading the public key) did not need the .cn at the end of the domain name, but the regionalized URLs did.